.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on: [push]
+
+
+jobs:
+  python-noconda:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          source .github/install.sh
+          python --version
+          pip install travis-sphinx codecov pytest-cov
```

Comment:
here

Reply:
ah, yes, we might have to use something other than travis-sphinx to deploy docs.  It's not really needed, if we add a seperate github action for doc deployment though.  It was only because travis didn't have graphs of actions.  Basically that action should only run if this is a merge to master commit and all CI actions pass

Reply:
do you know if we can also build the PR docs like on readthedocs?
I find that tool useful to check the docs without having to build them myself

Reply:
there is a sphinx action for github actions that even does rest linting: https://github.com/ammaraskar/sphinx-action

Reply:
done